neuromation/api/utils.py

```diff
@@ -112,3 +115,13 @@ async def retry() -> AsyncIterator[None]:
 
 def flat(sql: str) -> str:
     return " ".join(line.strip() for line in sql.splitlines() if line.strip())
+
+
+def find_project_root(cwd: Optional[Path] = None) -> Path:
```

Comment:
I am not sure `cwd` is good name.

Reply:
I agree that `path` is more appropriate here, thanks for suggestion.